### ISSUE TITLE
Tokenization

### ISSUE THREAD

**Issue by chledowski** [BIU-NLP/Breaking_NLI#1]
Hi!

Thanks for your nice work. I was wondering if you are using any kind of tokenization to deal with sentences like ''A man playing scrabble with carrots, celery, onions, limes, apples and cauliflower with another person.''
More precisely, I was wondering how are you dealing with commas and dots.

Thanks,
Kuba

**Comment by Max216** [comment]
Hi Kuba,

for the replacement part we did not use a tokeninzer but regular expressions (`$\b<term-to-replace>\b$`). However in the next step (when validating the sentences using bigrams), we used tokenized sentences and also discard them, if `<replaced-word>` does not exactly match a single token (or a sequence of accordingly matching tokens for multi-word expressions). Thus, we ensure that the replacements happen on proper tokens. I hope this answers your question :).

Best,
Max
